# Starting in glib 2.10 (released May 2009), the gettext() routine was added
# as part of POSIX 2008. 
# Since this clashes with local functions, they are renamed
diff -Naur tools/sir_modhead.c tools/sir_modhead.c
--- /tools/sir_modhead.c	2005-12-09 16:47:38.000000000 -0700
+++ /tools/sir_modhead.c	2012-10-22 16:15:21.000000000 -0600
@@ -33,7 +33,7 @@
 
 void Sarg_process(sir_head *out_head, int nSarg, char **Sarg_param, char **Sarg_value);
 
-int getline (FILE *imf, char *line, int max_length);
+int sirmodhead_getline (FILE *imf, char *line, int max_length);
 
 void list_header_variables(FILE *imf);
 
@@ -160,7 +160,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *d);
-  (void) getline(imf, line, MAXLINE);
+  (void) sirmodhead_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%d",d);
 }
 
@@ -169,7 +169,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *f);
-  (void) getline(imf, line, MAXLINE);
+  (void) sirmodhead_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%f",f);
 }
 
@@ -178,7 +178,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, s);
-  (void) getline(imf, line, MAXLINE);
+  (void) sirmodhead_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) strncpy(s,line,l);
 }
 
@@ -289,7 +289,7 @@
   fprintf(stdout,"                            12 = EASE South Hem,\n");
   fprintf(stdout,"                            13 = EASE Global\n");
   fprintf(stdout,"    IOPT: [Def=%d (%s)] ",head->iopt,ptype);
-  (void) getline(stdin, line, MAXLINE);  
+  (void) sirmodhead_getline(stdin, line, MAXLINE);  
   if (strlen(line) > 0) sscanf(line,"%d",&head->iopt);
   switch(head->idatatype){
   case -1:
@@ -369,13 +369,13 @@
   print_sir_head(stdout, head);    
 
   fprintf(stdout,"Accept as is or Revise (A/R) [default=A]:");
-  (void) getline(stdin, line, MAXLINE);
+  (void) sirmodhead_getline(stdin, line, MAXLINE);
   if (*line == 'R' || *line == 'r') goto LOOP;
 
 }
 
 
-int getline(FILE *imf, char *line, int max_length)
+int sirmodhead_getline(FILE *imf, char *line, int max_length)
 {  /* read a newline-terminated character string from a file stream */
   char *s, ch;
   int n=0;
diff -Naur tools/sir_remapper2.c tools/sir_remapper2.c
--- /tools/sir_remapper2.c	2009-04-13 14:19:44.000000000 -0600
+++ /tools/sir_remapper2.c	2012-10-22 16:15:21.000000000 -0600
@@ -67,7 +67,7 @@
 
 void Sarg_process(sir_head *out_head, int nSarg, char **Sarg_param, char **Sarg_value);
 
-int getline (FILE *imf, char *line, int max_length);
+int sir_remapper_getline (FILE *imf, char *line, int max_length);
 
 void list_header_variables(FILE *imf);
 
@@ -430,7 +430,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *d);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%d",d);
 }
 
@@ -439,7 +439,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *f);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%f",f);
 }
 
@@ -448,7 +448,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, s);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) strncpy(s,line,l);
 }
 
@@ -514,7 +514,7 @@
   fprintf(stdout,"                            12 = EASE South Hem,\n");
   fprintf(stdout,"                            13 = EASE Global\n");
   fprintf(stdout,"    IOPT: [Def=%d (%s)] ",head->iopt,ptype);
-  (void) getline(stdin, line, MAXLINE);  
+  (void) sir_remapper_getline(stdin, line, MAXLINE);  
   if (strlen(line) > 0) sscanf(line,"%d",&head->iopt);
   switch(head->idatatype){
   case -1:
@@ -596,13 +596,13 @@
   print_sir_head(stdout, head);    
 
   fprintf(stdout,"Accept as is or Revise (A/R) [default=A]:");
-  (void) getline(stdin, line, MAXLINE);
+  (void) sir_remapper_getline(stdin, line, MAXLINE);
   if (*line == 'R' || *line == 'r') goto LOOP;
 
 }
 
 
-int getline(FILE *imf, char *line, int max_length)
+int sir_remapper_getline(FILE *imf, char *line, int max_length)
 {  /* read a newline-terminated character string from a file stream */
   char *s, ch;
   int n=0;
@@ -1063,10 +1063,10 @@
 
   /* skip header lines */
 
-  (void) getline(imf, line, MAXLINE);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
 
-  while (end_flag == 0 && getline(imf,line, MAXLINE) > 0) {
+  while (end_flag == 0 && sir_remapper_getline(imf,line, MAXLINE) > 0) {
     /* printf("%s\n",line); */
     sscanf(line,"%d %d %d %f %f %f %f %s", &reg, &prj, &DtLn, &LLlat, &LLlon, &URlat, &URlon, regname);
     
diff -Naur tools/sir_remapper3.c tools/sir_remapper3.c
--- /tools/sir_remapper3.c	2005-12-09 16:47:38.000000000 -0700
+++ /tools/sir_remapper3.c	2012-10-22 16:15:21.000000000 -0600
@@ -68,7 +68,7 @@
 
 void Sarg_process(sir_head *out_head, int nSarg, char **Sarg_param, char **Sarg_value);
 
-int getline (FILE *imf, char *line, int max_length);
+int sir_remapper_getline (FILE *imf, char *line, int max_length);
 
 void list_header_variables(FILE *imf);
 
@@ -449,7 +449,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *d);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%d",d);
 }
 
@@ -458,7 +458,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *f);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%f",f);
 }
 
@@ -467,7 +467,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, s);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) strncpy(s,line,l);
 }
 
@@ -533,7 +533,7 @@
   fprintf(stdout,"                            12 = EASE South Hem,\n");
   fprintf(stdout,"                            13 = EASE Global\n");
   fprintf(stdout,"    IOPT: [Def=%d (%s)] ",head->iopt,ptype);
-  (void) getline(stdin, line, MAXLINE);  
+  (void) sir_remapper_getline(stdin, line, MAXLINE);  
   if (strlen(line) > 0) sscanf(line,"%d",&head->iopt);
   switch(head->idatatype){
   case -1:
@@ -615,13 +615,13 @@
   print_sir_head(stdout, head);    
 
   fprintf(stdout,"Accept as is or Revise (A/R) [default=A]:");
-  (void) getline(stdin, line, MAXLINE);
+  (void) sir_remapper_getline(stdin, line, MAXLINE);
   if (*line == 'R' || *line == 'r') goto LOOP;
 
 }
 
 
-int getline(FILE *imf, char *line, int max_length)
+int sir_remapper_getline(FILE *imf, char *line, int max_length)
 {  /* read a newline-terminated character string from a file stream */
   char *s, ch;
   int n=0;
@@ -1082,10 +1082,10 @@
 
   /* skip header lines */
 
-  (void) getline(imf, line, MAXLINE);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
 
-  while (end_flag == 0 && getline(imf,line, MAXLINE) > 0) {
+  while (end_flag == 0 && sir_remapper_getline(imf,line, MAXLINE) > 0) {
     /* printf("%s\n",line); */
     sscanf(line,"%d %d %d %f %f %f %f %s", &reg, &prj, &DtLn, &LLlat, &LLlon, &URlat, &URlon, regname);
     
@@ -1476,10 +1476,10 @@
 
   /* skip header lines */
 
-  (void) getline(imf, line, MAXLINE);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
 
-  while (end_flag == 0 && getline(imf,line, MAXLINE) > 0) {
+  while (end_flag == 0 && sir_remapper_getline(imf,line, MAXLINE) > 0) {
     sscanf(line,"%d %d %d %f %f %f %f %s", &reg, &prj, &DtLn, &LLlat, &LLlon, &URlat, &URlon, regname);    
     if (reg == region_num)
       end_flag = 1;
diff -Naur tools/sir_remapper.c tools/sir_remapper.c
--- /tools/sir_remapper.c	2005-12-09 16:47:38.000000000 -0700
+++ /tools/sir_remapper.c	2012-10-22 16:15:21.000000000 -0600
@@ -53,7 +53,7 @@
 
 void Sarg_process(sir_head *out_head, int nSarg, char **Sarg_param, char **Sarg_value);
 
-int getline (FILE *imf, char *line, int max_length);
+int sir_remapper_getline (FILE *imf, char *line, int max_length);
 
 void list_header_variables(FILE *imf);
 
@@ -363,7 +363,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *d);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%d",d);
 }
 
@@ -372,7 +372,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, *f);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) sscanf(line,"%f",f);
 }
 
@@ -381,7 +381,7 @@
   char line[MAXLINE];
 
   fprintf(stdout,prompt, s);
-  (void) getline(imf, line, MAXLINE);
+  (void) sir_remapper_getline(imf, line, MAXLINE);
   if (strlen(line) > 0) strncpy(s,line,l);
 }
 
@@ -478,7 +478,7 @@
   fprintf(stdout,"                            12 = EASE South Hem,\n");
   fprintf(stdout,"                            13 = EASE Global\n");
   fprintf(stdout,"    IOPT: [Def=%d (%s)] ",head->iopt,ptype);
-  (void) getline(stdin, line, MAXLINE);  
+  (void) sir_remapper_getline(stdin, line, MAXLINE);  
   if (strlen(line) > 0) sscanf(line,"%d",&head->iopt);
   switch(head->idatatype){
   case -1:
@@ -560,13 +560,13 @@
   print_sir_head(stdout, head);    
 
   fprintf(stdout,"Accept as is or Revise (A/R) [default=A]:");
-  (void) getline(stdin, line, MAXLINE);
+  (void) sir_remapper_getline(stdin, line, MAXLINE);
   if (*line == 'R' || *line == 'r') goto LOOP;
 
 }
 
 
-int getline(FILE *imf, char *line, int max_length)
+int sir_remapper_getline(FILE *imf, char *line, int max_length)
 {  /* read a newline-terminated character string from a file stream */
   char *s;
   int ch;
